src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,81 @@
-export const Pagination = () => {};
+import React from 'react';
+
+type Props = {
+  // total: number,
+  visibleItems: string[],
+  perPage: number,
+  currentPage: number,
+  onPageClick: (page: number) => void,
+  toPrevPage: () => void,
+  toNextPage: () => void,
+};
+
+export const Pagination: React.FC<Props> = ({
+  visibleItems,
+  perPage,
+  currentPage,
+  onPageClick,
+  toPrevPage,
+  toNextPage,
+}) => {
+  const arrayOfPages = Array.from(Array(perPage), (_, i) => i + 1);
+
+  return (
+    <>
+      <ul className="pagination">
+        <li className={`page-item ${currentPage === arrayOfPages[0] && 'disabled'}`}>
+          <a
+            data-cy="prevLink"
+            className="page-link"
+            href="#prev"
+            aria-disabled={currentPage === arrayOfPages[0]}
```

Comment:
Move all repeatable values to variables. Here is a `currentPage === arrayOfPages[0]` value for example